### PR TITLE
Add create notification function

### PR DESCRIPTION
**Pull request by DiegoCuevas**

### CODE REVIEW

**Review by AngieCristina — DISMISSED** [review]

File: Api/app/controllers/notifications_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class NotificationsController < ApplicationController
   def index
-    render json: { has_notifications: current_user.notify_user.where(status: "Unread").size > 0 }
+    render json: Notification.all
+    # render json: { has_notifications: current_user.notify_user.where(status: "Unread").size > 0 }
```
Comment (MariethPerez) [inline Api/app/controllers/notifications_controller.rb]: I think that this line shouldn't be commented.

File: Api/app/models/request.rb [hunk 2]
```diff
@@ -15,10 +17,13 @@ class Request < ApplicationRecord
     rejected: "Rejected",
```
Comment (AngieCristina) [inline Api/app/models/request.rb:16]: Can you tell me wh you added just these two `rejected,` and `cancel` objects?

**Review by AngieCristina — DISMISSED** [review]

**Review by MariethPerez — DISMISSED** [review]

**Review by MariethPerez — APPROVED** [review]

File: Api/app/models/request.rb (hunk 2 above)
Comment (DiegoCuevas) [reply Api/app/models/request.rb:16]: It's a little mistake but it doesn't affect anything else because I don't call the variable.

Comment (AngieCristina) [reply Api/app/models/request.rb:16]: Ok, you should fix it in the next PR, thank you

**Review by AngieCristina — APPROVED** [review]

**Pull request merged**